posthog/hogql/escape_sql.py

```diff
@@ -30,10 +30,10 @@ def escape_param_clickhouse(value: str) -> str:
     return "'%s'" % "".join(singlequote_escape_chars_map.get(c, c) for c in str(value))
 
 
-# Copied from clickhouse_driver.util.escape, adapted from single quotes to backquotes. Added a $.
+# Copied from clickhouse_driver.util.escape, adapted from single quotes to backquotes. Added a `$` and a `:`.
```

Comment:
I also ran into this with characters "/" (e.g. feature flag variant properties) and "-" (just all sorts of properties), so I think we need a more general solution, such as a `foo['bar]'` syntax.

Reply:
The current syntax for this seems to be `properties."abc:def"`, which is necessary in this case even with this fix, as otherwise the "." will work as nested property accessor. Should I add "/" and "-" to the identifier?